app/models/orphan.rb

```diff
@@ -175,16 +183,14 @@ def orphan_born_long_after_fathers_death?
   end
 
   def can_be_inactivated
-    if currently_sponsored?
-      errors.add :orphan_status,
+    if sponsored? && being_inactivated?
```

Comment:
I moved the `being_inactivated?` conditional here from the `validate` call on L74 as it makes the code more readable. Before, the method said "If sponsored, then add an error" and one would have to go back to the `validate` call to see that the method would only be called when an orphan is being inactivated. Now it says "If sponsored and being inactivated, then add an error", which can be understood in isolation. Thoughts?